app/models/transac.rb

```diff
@@ -0,0 +1,6 @@
+class Transac < ApplicationRecord
+  belongs_to :category
+
+  validates :amount, presence: true
+  # has_many :categories, dependent: :destroy
```

Comment:
- [ ] Here as well. I would recommend removing the commented code to make the codebase cleaner.